python/taichi/misc/gui.py

```diff
@@ -102,21 +121,60 @@ def value(self, value):
             self.gui.core.set_widget_value(self.wid, value)
 
     def slider(self, text, minimum, maximum, step=1):
+        """Create a slider object on canvas to be manipulated with.
+
+        Args:
+            text (str): The title of slider.
+            minimum (Number): The minimum value of slider.
+            maximum (Number): The maximum value of slider.
+            step (Number, optional): The changing step of slider. Default is 1.
+
+        Return:
+            The created slider object.
```

Comment:
Add the type of return value?

e.g.:
```suggestion
        :class:`~taichi.misc.gui.GUI.WidgetValue` : The created slider object.
```

Reply:
Got it. I'll work on this.